src/libstd/sys/windows/fs.rs

```diff
@@ -567,23 +647,85 @@ pub fn remove_dir_all(path: &Path) -> io::Result<()> {
         // rmdir only deletes dir symlinks and junctions, not file symlinks.
         rmdir(path)
     } else {
-        remove_dir_all_recursive(path)
+        // canonicalize to get a `/??/`-path, which can handle files like `CON`
```

Comment:
`\\?\`, not `/??/`. We're canonicalizing to verbatim paths not NT paths which are internal to the kernel.